content/soarian-financials/stu3/individuals/patient.md

```diff
@@ -0,0 +1,638 @@
+	"reference": "Organization/900000",
+	"display": "System Id Issuer"
+	}
+	},
+	{
+	"type": {
+	"coding": [
+	{
+	"system": "http://hl7.org/fhir/v2/0203",
+	"code": "MR",
+	"display": "Medical Record Number"
+	}
+	],
+	"text": "Medical Record Number"
+	},
+	"value": "11080323",
+	"period": {
+	"start": "2018-09-27"
+	},
+	"assigner": {
+	"reference": "Organization/28",
+	"display": "EM Health System"
+	}
+	},
+	{
+	"type": {
+	"coding": [
+	{
+	"system": "http://hl7.org/fhir/v2/0203",
+	"code": "SS",
+	"display": "Social Security Number"
+	}
+	],
+	"text": "Social Security Number"
+	},
+	"value": "547487444",
+	"period": {
+	"start": "2018-12-14"
+	},
+	"assigner": {
+	"reference": "Organization/900001",
+	"display": "U. S. Government"
+	}
+	},
+	{
+	"type": {
+	"coding": [
+	{
+	"system": "http://hl7.org/fhir/v2/0203",
+	"code": "MR",
+	"display": "Medical Record Number"
+	}
+	],
+	"text": "Medical Record Number"
+	},
+	"value": "254147",
+	"period": {
+	"start": "2018-12-14"
+	},
+	"assigner": {
+	"reference": "Organization/2342",
+	"display": "Steamy Falls IP Hosp"
+	}
+	},
+	{
+	"type": {
+	"coding": [
+	{
+	"system": "http://hl7.org/fhir/v2/0203",
+	"code": "DL",
+	"display": "Drivers License"
+	}
+	],
+	"text": "Drivers License"
+	},
+	"value": "25447457",
+	"period": {
+	"start": "2018-12-14"
+	},
+	"assigner": {
+	"reference": "Organization/118",
+	"display": "Commonwealth of PA"
+	}
+	}
+	],
+	"active": true,
+	"name": [
+	{
+	"use": "OFFICIAL",
+	"text": "James,Smith,Edward",
+	"family": "James",
+	"given": [
+	"Smith",
+	"Edward"
+	],
+	"period": {
+	"start": "2018-12-14"
+	}
+	}
+	],
+	"telecom": [
+	{
+	"extension": [
+	{
+	"valueBoolean": true,
+	"url": "http://hl7.org/fhir/StructureDefinition/iso21090-preferred"
+	}
+	],
+	"system": "PHONE",
+	"value": "(645) 748 4747",
+	"use": "HOME"
+	},
+	{
+	"extension": [
+	{
+	"valueBoolean": false,
+	"url": "http://hl7.org/fhir/StructureDefinition/iso21090-preferred"
+	}
+	],
+	"system": "PHONE",
+	"value": "(547) 457 4848",
+	"use": "MOBILE"
+	},
+	{
+	"system": "EMAIL",
+	"value": "james@cerner.com"
+	}
+	],
+	"gender": "MALE",
+	"birthDate": "2008-12-11",
+	"deceasedBoolean": false,
+	"address": [
+	{
+	"use": "HOME",
+	"type": "POSTAL",
+	"text": "51 Valley Stream parkway,Malvern,PA 19355",
+	"line": [
+	"51 Valley Stream parkway"
+	],
+	"city": "Malvern",
+	"district": "Chester",
+	"state": "PA",
+	"postalCode": "19355",
+	"country": "US",
+	"period": {
+	"start": "2018-12-14"
+	}
+	}
+	],
+	"maritalStatus": {
+	"coding": [
+	{
+	"system": "http://hl7.org/fhir/v3/MaritalStatus",
+	"code": "S",
+	"display": "Single"
+	}
+	],
+	"text": "Single"
+	},
+	"communication": [
+	{
+	"language": {
+	"coding": [
+	{
+	"system": "urn:ietf:bcp:47",
+	"code": "en",
+	"display": "ENGLISH"
+	}
+	],
+	"text": "ENGLISH"
+	},
+	"preferred": true
+	}
+	]
+	}
+
+### Errors
+
+The following common errors and OperationOutcomes may be returned.
+
+Error														 | Severity   | Code 			  | Diagnostics
```

Comment:
keep a leading space for table header/entry rows. delete empty columns. keep a trailing dash at the end of the table. use spaces not tabs

i.e. this changes to
```
 Error                                                  | Severity | Code          | Diagnostics
--------------------------------------------------------|----------|---------------|---------------------------------------------------------------------------------------------------------
 There is no active  legal name                         | Error    | not-supported | Retrieval of unidentified patient is not supported at this time.
 The patient being retrieved is merged                  | Error    | not-supported | The patient being searched for has been merged. The merged target patient number is (Target Person No).
 The patient being retrieved is not found in the system | Error    | not-found     | The patient is not found.
```